src/test/java/htsjdk/variant/vcf/VCFCodec43FeaturesTest.java

```diff
@@ -0,0 +1,258 @@
+package htsjdk.variant.vcf;
+
+import htsjdk.samtools.util.CloseableIterator;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.TestUtil;
+import htsjdk.samtools.util.Tuple;
+import htsjdk.tribble.index.Index;
+import htsjdk.tribble.index.IndexFactory;
+import htsjdk.tribble.util.TabixUtils;
+import htsjdk.variant.VariantBaseTest;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/*
+ **************************************************************************************************
+ * IMPORTANT NOTE: this class contains string constants that contain embedded non-ASCII characters
```

Comment:
I'm worried that this will cause problems similar to https://github.com/broadinstitute/gatk/issues/5934.  I think we might want to figure out how to fix the encoding in the build.gradle so it doesn't use system specific encodings during build.  

I think we can probably fix it with something like 
```
 tasks.withType(JavaCompile) {
   options.encoding = "UTF-8"
 }

 tasks.withType(Javadoc) {
    options.addStringOption('encoding', 'UTF-8')
 }
```  
added to build.gradle but I haven't  reproduced the issue locally to test yet.